exoscale_auth.py

```diff
@@ -29,3 +31,62 @@ def __call__(self, request):
             'Authorization': auth,
         })
         return request
+
+
+class ExoscaleV2Auth(AuthBase):
+    def __init__(self, key, secret):
+        self.key = key
+        self.secret = secret.encode('utf-8')
+
+    def __call__(self, request):
+        expiration = datetime.now(tz=timezone.utc) + timedelta(minutes=10)
```

Comment:
```suggestion
        expiration = int(time.time()) + 10 * 60
```

It'd be fine if `_sign_request` accepts a timestamp directly - it's easy enough to generate with `time.time()`

Reply:
Changed in d159467.